test/integration/attachment_replacement_integration_test.rb

```diff
@@ -66,7 +66,7 @@ class AttachmentReplacementIntegrationTest < ActionDispatch::IntegrationTest
           stub_publishing_api_has_linkables([], document_type: "topic")
           visit admin_news_article_path(edition)
           click_button "Create new edition to edit"
-          click_link "Attachments 1"
+          visit admin_edition_attachments_path(edition.latest_edition)
```

Comment:
Not something that can be solved in this ticket, but it makes me sad that we need to directly navigate to the attachments page. 😢 

Ideally this is the sort of test that'd navigate as if it were a real user (e.g. click links). But that's not possible here because there's no user friendly way to navigate from the 'edit' screen to the 'summary' and on to 'attachments'.

Reply:
I've actually done this for the other specs. Not sure why i was lazy here. Will update now!